Python/gc_free_threading.c

```diff
@@ -439,7 +439,10 @@ process_delayed_frees(PyInterpreterState *interp)
 static int
 visit_decref(PyObject *op, void *arg)
 {
-    if (_PyObject_GC_IS_TRACKED(op) && !_Py_IsImmortal(op)) {
+    if (_PyObject_GC_IS_TRACKED(op)
+        && !_Py_IsImmortal(op)
+        && (op->ob_gc_bits & _PyGC_BITS_FROZEN) == 0
+    ) {
```

Comment:
```suggestion
    if (_PyObject_GC_IS_TRACKED(op) &&
       !_Py_IsImmortal(op) &&
       (op->ob_gc_bits & _PyGC_BITS_FROZEN) == 0) {
```
(suggestion :) )

Reply:
No, we changed this in PEP 7 recently: https://discuss.python.org/t/pep-7-break-lines-before-operators-like-pep-8/62402

Reply:
Hmmm, It seems I'm too old-fashioned :(